docs/user/integration/prometheus/README.md

```diff
@@ -0,0 +1,154 @@
+    ```
+1. If you haven't created the Namespace yet, now is the time to do so:
+    ```bash
+    kubectl create namespace $K8S_PROM_NAMESPACE
+    ```
+   >**Note**: This Namespace must have **no** Istio sidecar injection enabled; that is, there must be no `istio-injection` label present on the Namespace. The Helm chart deploys jobs that will not succeed when Isto sidecar injection is enabled.
+
+1. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Run the following command:
+    ```bash
+    export HELM_PROM_RELEASE="prometheus"
+    ```
+
+1. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+    ```
+
+1. Run the Helm upgrade command, which installs the chart if it's not present yet. At the end of the command, change the Grafana admin password to some value of your choice.
+    ```bash
+    helm upgrade --install -n ${K8S_PROM_NAMESPACE} ${HELM_PROM_RELEASE} prometheus-community/kube-prometheus-stack -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/prometheus/values.yaml --set grafana.adminPassword=myPwd
+    ```
+
+1. You can use the [values.yaml](./values.yaml) provided with this tutorial, which contains customized settings deviating from the default settings, or create your own one.
+The provided `values.yaml` covers the following adjustments:
+- Basic Istio setup to secure communication between Prometheus, Grafana and Alertmanager
+- Native OTLP receiver enabled for prometheus
+- Basic configuration of data persistence with retention
+- Basic resource limits for involved components
+
+## Verify the kube-prometheus-stack
+
+1. You should see several Pods coming up in the Namespace, especially Prometheus, Grafana and Alertmanager. Assure that all Pods have the "Running" state.
+2. Browse the Prometheus dashboard and verify that all "Status->Targets" are healthy. The following command exposes the dashboard on `http://localhost:9090`:
+   ```bash
+   kubectl -n ${K8S_PROM_NAMESPACE} port-forward $(kubectl -n ${K8S_PROM_NAMESPACE} get service -l app=kube-prometheus-stack-prometheus -oname) 9090
+   ```
+3. Browse the Grafana dashboard and verify that the dashboards are showing data. The user `admin` is preconfigured in the Helm chart; the password was provided in your `helm install` command. The following command exposes the dashboard on `http://localhost:3000`:
+   ```bash
+   kubectl -n ${K8S_PROM_NAMESPACE} port-forward svc/${HELM_PROM_RELEASE}-grafana 3000:80
+   ```
+
+## Activate a MetricPipeline
+
+1. Apply a MetricPipeline resource having the output configured with the local prometheus URL and having the inputs enabled for collecting Istio metrics and collecting application metrics whose workloads are annotated with prometheus annotations.
+```yaml
```

Comment:
I believe you need 3 blank spaces at the beginning of each line for indentation under the step

Reply:
fixed